.github/workflows/ci.yml

```diff
@@ -22,9 +22,9 @@ jobs:
     - uses: actions/setup-go@v5
       with:
         go-version: ${{ matrix.go-version }}
-    - uses: goreleaser/goreleaser-action@v2
+    - uses: goreleaser/goreleaser-action@v6
       with:
-        args: release
+        args: release --snapshot
```

Comment:
Thank you for looking into this. I wonder if the `--snapshot` flag is always needed or only when we don't have a tag. When we have a tag, the flag should likely be left out, so goreleaser generates a tagged release.

Reply:
Sounds correct to me.
* `--snapshot` is appropriate if no tag is provided.
* If a tag is provided then `--snapshot` should not be used.

Reply:
```suggestion
        args: ${{ github.ref_type == 'tag' && 'release' || 'release --snapshot' }}
```